aips/aip-1.md

```diff
@@ -0,0 +1,65 @@
+---
+aip: (this is determined by the AIP Manager)
+title: Proposer selection improvements
+author: igor-aptos
+discussions-to (*optional): TBD
+Status: Draft
+last-call-end-date (*optional):
+type: Standard (Core)
+created: 12/7/2022
+updated (*optional):
+requires (*optional): <AIP number(s)>
```

Comment:
for optional fields. if you don't have anything here you can just delete those two fields